webpack/published-ts-components.js

```diff
@@ -103,8 +103,6 @@ const publishedTSComponents = {
   useActiveDescendantListFocus: "hooks/useActiveDescendantListFocus",
   useListenFocusTriggers: "hooks/useListenFocusTriggers",
   useSwitch: "hooks/useSwitch",
-  // Don't remove next line
```

Comment:
are you sure? :)

Reply:
Pretty sure
We have this marker on the line 87